bin/run-semgrep.sh

```diff
@@ -0,0 +1,2 @@
+export SEMGREP_RULES="p/default p/expressjs p/react"
```

Comment:
it's better to find some extra rules for us

Reply:
done